5-setup-loki-stack/res/yamls/loki-values.yaml

```diff
@@ -0,0 +1,84 @@
+loki:
+  enabled: true
+
+promtail:
+  enabled: true
+
+fluent-bit:
+  enabled: false
+
+grafana:
+  enabled: false
+  sidecar:
+    datasources:
+      enabled: true
+  image:
+    tag: 7.5.0
+
+prometheus:
+  enabled: false
+
+filebeat:
+  enabled: false
+  filebeatConfig:
+    filebeat.yml: |
+      # logging.level: debug
+      filebeat.inputs:
+      - type: container
+        paths:
+          - /var/log/containers/*.log
+        processors:
+        - add_kubernetes_metadata:
+            host: ${NODE_NAME}
+            matchers:
+            - logs_path:
+                logs_path: "/var/log/containers/"
+      output.logstash:
+        hosts: ["logstash-loki:5044"]
+
+logstash:
+  enabled: false
+  image: grafana/logstash-output-loki
+  imageTag: 1.0.1
+  filters:
+    main: |-
+      filter {
+        if [kubernetes] {
+          mutate {
+            add_field => {
+              "container_name" => "%{[kubernetes][container][name]}"
+              "namespace" => "%{[kubernetes][namespace]}"
+              "pod" => "%{[kubernetes][pod][name]}"
+            }
+            replace => { "host" => "%{[kubernetes][node][name]}"}
+          }
+        }
+        mutate {
+          remove_field => ["tags"]
+        }
+      }
+  outputs:
+    main: |-
+      output {
+        loki {
+          url => "http://loki:3100/loki/api/v1/push"
+          #username => "test"
+          #password => "test"
+        }
+        # stdout { codec => rubydebug }
+      }
+
+storage_config:
+  aws:
+    bucketnames: loki-starterkit-storage
+    endpoint: fra1.digitaloceanspaces.com
+    region: fra1
+    access_key_id: 5NPIUJ7DPF4B5THD3TK5
+    secret_access_key: CetB1roHmBuLYtT4XxlO0ieP+5Q9GdB114vcjIcKd6s
```

Comment:
Using real secret keys in tutorial examples or anywhere in public is totally forbidden. Please use `<>` placheholders instead.  

Example:
```
access_key_id: <DO_SPACES_ACCESS_KEY_ID>
secret_access_key=<DO_SPACES_SECRET_ACCESS_KEY>
``` 
Then, guide the user to replace the placeholders accordingly.